wikimedia/wikipedia-ios
# Use feed refresh as fallback for Picture of the day widget

**Phabricator:** https://phabricator.wikimedia.org/T262358

### Notes

Adds use of `updateFeedSourcesUserInitiated` as network fallback if user's data store doesn't contain current day's picture. Also updates the previously erroneous use of `memoryCachedImage` to `cachedImage` to expand pool of available cached data.

### Test Steps
Similar to https://github.com/wikimedia/wikipedia-ios/pull/3685. In general, if network connectivity is available, the placeholder 2019 entry should not be visible after the POTD widget is added to the home screen or today view.
